example/node-pools/main.tf

```diff
@@ -0,0 +1,152 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.48.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "=2.3.0"
+    }
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "=1.13.0"
```

Comment:
Should we be using version 2 of the Kubernetes provider now its gone GA?

Reply:
we actually didn't need it, removed.